helm/csi-isilon/values.yaml

```diff
@@ -59,6 +59,16 @@ kubeletConfigDir: /var/lib/kubelet
 # Default value: false
 enableCustomTopology: false
 
+# fsGroupPolicy: This is defined in Driver's spec. Supports three different mode which defines volume modification and ownership change upon mount operation.
```

Comment:
Can you rephrase this sentence to: "Defines if the underlying volume supports changing ownership and permission of the volume before being mounted."

Reply:
done